.github/workflows/qiita-ci.yml

```diff
@@ -234,4 +234,4 @@ jobs:
     - name: lint
       run: |
         pip install -q flake8
-        flake8 qiita_* setup.py scripts/qiita*
+        flake8 qiita_* setup.py scripts/qiita* notebooks/*/*.py
```

Comment:
Just to confirm, the notebooks directory is a new directory that will contain just the .py file below? Just checking to confirm that your local branch doesn't have additional files you want added. Also, this will not check any ipynb notebooks. This is all right yes?

Reply:
Correct.